command.go

```diff
@@ -156,13 +154,16 @@ func (c *Command) Get(path []string) (*Command, error) {
 
 // GetOptions returns the options in the given path of commands
 func (c *Command) GetOptions(path []string) (map[string]cmdkit.Option, error) {
+	if c == nil {
```

Comment:
when would we encounter this?

Reply:
Whenever we create a new request with a nil root command. Especially during testing it's handy to have the thing behave normal (or at least doesn't crash), even if you don't initialize every detail.    But yeah, maybe it's better not to allow this to avoid silent errors. I had to add an empty command in some tests after removing the check, but they pass again.